ntbs-service/Migrations/20210429162156_DefaultDrugResistanceProfile.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class DefaultDrugResistanceProfile : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                name: "Species",
+                table: "DrugResistanceProfile",
+                type: "nvarchar(30)",
+                maxLength: 30,
+                nullable: true,
```

Comment:
You've certainly fulfilled the scope of the ticket, but I was wondering if we want to keep these fields nullable? Can they be set to NULL when importing a notification? And is there any other meaning of NULL for existing records other than "No result"?

Reply:
Yeah there is no good reason for them to be nullable. During migrations these start off as being null and then get populated, but there is no reason why they couldn't start off as `'No result'` and then get populated. I'm adding a migration for this now